### DISCUSSION TITLE
Hydration error (SSR)

### DISCUSSION THREAD

**Discussion by sangelxyz** [leptos-rs/leptos#3003] [Q&A]
I have a basic component that's causing a hydration error, the error is A Left, DIV right. removing the links causes no errors. 
This only happens in debug mode, compiled it runs fine. It's also not inside a Suspense but i do use them where this happens.

Is there any issues with this code that could cause this

`#[component]
pub fn FooterList() -> impl IntoView {
    view! {
        <div class="footer">
            <div>
                <div>not trading advice. just data. always do your own research.</div>
            </div>
            <div>
                <div><a href="/rules/">rules</a></div>
                <div><a href="/privacy-policy/">privacy policy</a></div>
                <div><a href="/cookies/">cookies</a></div>
                <div><a href="/disclaimer/">Disclaimer</a></div>
                <div><a href="/glossary/">Glossary</a></div>
                <div><a href="/terms-of-use/">Terms of Use</a></div>
            </div>
            <div>
                <span>Suggestions / Praise / Problems to </span>
                <span><a href="mailto:">email</a></span>
            </div>
        </div>
    }
}`

**Comment by gbj** [comment]
It is likely an issue *not* with this code -- for example, if the footer is after something else that didn't hydrate correctly, the first issue might appear with the footer itself.

I just copy-pasted your `FooterList` into the end of a working example and it did not cause a hydration issue, so I do not think this component itself is the problem.

**Reply by sangelxyz** [reply]
Thanks for that, i'll go though my code gives me a chance to refactor it.
